tools/compare_v_performance_between_commits

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -e 
+
+if (( $# != 2 )); then
+    echo "Usage: compare_v_to_c_performance COMMIT_BEFORE COMMIT_AFTER"
```

Comment:
```sh
printf '%s\n' "Usage: compare_v_to_c_performance COMMIT_BEFORE COMMIT_AFTER"
```
(echo is really badly defined and thus discouraged from being used in cross-platform scripts)
An alternative I've used in older scripts I didn't want to completely rewrite:
```sh
echo(){ printf '%s\n' "$*"; }
```
I won't comment on `echo` from this onwards.

Reply:
echo is a bash builtin command, but ok